src/test/resources/application.yaml

```diff
@@ -4,4 +4,6 @@ spring:
     username: testUsername
     password: testPassword
     driver-class-name: org.postgresql.Driver
-    platform: postgres
\ No newline at end of file
+    platform: postgres
+  jpa:
+    show-sql: true
```

Comment:
That's the place where we actually need a blanc line. Please, add one in the end of the file